tests/cover/test_type_lookup.py

```diff
@@ -28,7 +28,7 @@
 # Build a set of all types output by core strategies
 blacklist = [
     'builds', 'iterables', 'permutations', 'random_module', 'randoms',
-    'runner', 'sampled_from', 'streaming',
+    'runner', 'sampled_from', 'streaming', 'fspaths',
```

Comment:
To be clear, `fspaths` *really does* belong on the blacklist - it has multiple output types by design.  

The registry addition is unrelated except that this line is what reminded me of it - though the combination disallow-pathlike/map-pathlike I suggest for the registry is specifically to ensure that we always get a pathlike object.